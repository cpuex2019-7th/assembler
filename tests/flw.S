    li t1, 0x3f666666           ; 0.9
    sw t1, sp, 0
    
    flw ft0, sp, 0
    fmvxw a0, ft0               ; a0 should be 0x3f666666
