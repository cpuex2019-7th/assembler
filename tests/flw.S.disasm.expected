lui t1, 259686
addi t1, t1, 1638
sw t1, sp, 0
flw ft0, sp, 0
fmvxw a0, ft0
